server/api/attendance/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var express = require('express');
+var controller = require('./attendance.controller');
+var config = require('../../config/environment');
+var auth = require('../../auth/auth.service');
+
+var router = express.Router();
+
+router.get('/', auth.isAuthenticated(), controller.index);
+router.get('/:id', auth.isAuthenticated(), controller.show);
+router.get('/present/me', auth.isAuthenticated(), controller.getAttendanceMe);
+router.get('/present/me/:date', auth.isAuthenticated(), controller.presentMe);
+router.post('/attend', auth.isAuthenticated(), controller.attend);
+
+router.post('/', auth.hasRole('mentor') || auth.hasRole('admin'), controller.create);
```

Comment:
as stated verbally. no need to explicitly allow `auth.hasRole('admin')` admin inherit all lower auth levels and can do anything a mentor can do. These should all be changed to `router.post('/', auth.hasRole('mentor'), controller.create);
 `